Makefile

```diff
@@ -235,7 +239,12 @@ install: $(buildbin)
 # the bin between deb/rpm/tar packages over building directly into the package
 # directory.
 $(buildbin):
+	echo $(GOOS)
 	@mkdir -pv $(dir $@)
+	if [ $(GOOS) = "windows" ]; then \
+		go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.4.0; \
```

Comment:
Having to have a go install command of a repo with a specific version in four places is less than ideal. Is this something that we maybe only do on releases?

Reply:
Definitely not ideal, moved the dependency installation to a shell script so its in one place.

Reply:
awesome